src/setup/devnet/templates/remote/bor/bor-config.toml.njk

```diff
@@ -0,0 +1,193 @@
+chain = "/var/lib/bor/genesis.json"
+identity = "MyIdentity"
+verbosity = 3
+log-level = ""
+vmdebug = false
+datadir = "/var/lib/bor/data"
+ancient = ""
+"db.engine" = "leveldb"
+keystore = "/var/lib/bor/keystore"
+"rpc.batchlimit" = 100
+"rpc.returndatalimit" = 100000
+syncmode = "full"
+gcmode = "full"
```

Comment:
How do we support the archive node currently?
Asking because if we just add an additional flag to the start.sh file, then we don't need this, because the flag will overwrite this value.

Reply:
We have that in .env right?
We should make sure to keep it in one place only

Reply:
Updated regarding this [here](https://github.com/maticnetwork/matic-cli/pull/215/commits/2b754a41f7812257800f4364717f71c784ddd4f0).